src/ResourceManagement/WebSite/Microsoft.Azure.Management.Websites/project.json

```diff
@@ -0,0 +1,54 @@
+{
+  "version": "1.0.0-preview",
```

Comment:
@naveedaz Is it okay we bump up the version here from 0.16.8-preview to 1.0.0-preview?